camelot/view/action_steps/application.py

```diff
@@ -236,25 +232,6 @@ def __post_init__(self, model_context, actions_state):
                 action_route = AdminRoute._register_list_action_route(model_context.admin.get_admin_route(), action)
                 self.action_states.append((action_route, state._to_dict()))
 
-    @classmethod
-    def gui_run(cls, gui_context_name, serialized_step):
```

Comment:
Hebben we dit niet meer nodig voor de forms?

Reply:
ik denk het niet, op de cpp model zit er een 'actionStateChanged' signal waarnaar de formview luistert om zijn action states te updaten